base/shared/src/main/scala/scalaz/ct/profunctor.scala

```diff
@@ -7,6 +7,10 @@ trait ProfunctorClass[F[_, _]] {
   def lmap[A, B, C](fab: F[A, B])(ca: C => A): F[C, B]
```

Comment:
can you rename this to `lcontramap` or something while you're at it?

Reply:
Sure